scripts/export_database.pl

```diff
@@ -394,7 +394,12 @@ sub sanitize_field_content {
 			$nid =~ s/-$//g;
 
 			if (defined $product_ref->{nutriments}{$nid . "_100g"}) {
-			$csv .= $product_ref->{nutriments}{$nid . "_100g"} . "\t";
+				my $value = $product_ref->{nutriments}{$nid . "_100g"};
+				if ($value =~ /e/) {
+					# 7e-05 1.71e-06
+					$value = sprintf("%.10f", $value);
```

Comment:
Are we really need 9 decimal digits? Where are the unit tests ;-)?

Reply:
We do need many decimal digits for some nutrients that are in micrograms. e.g.https://world.openfoodfacts.org/product/0746742290308/sparkling-water-beverage-with-juice-black-bear

We have 7.5e-08. That's 0.000000075

Reply:
There are no unit tests for the CSV export, we can add some, but it will take time and it should be a different bug.

Reply:
Can't we just add as decimals as we need for a specific number?

Eg: `1.71e-02 -> 0.0171`, `1.71e-06 -> 0,00000171`

From [this page](https://perldoc.perl.org/functions/sprintf.html) in perldocs:
```perl
printf '<%.1f>', 1;  # prints "<1.0>
```

Reply:
@VaiTon : it's possible, I'm just not sure it's worth risking introducing a bug while doing it. :)
e.g. we could use a regex to try to remove trailing 0s, or we could try to parse the 1.71e-02 to know that we need 4 digits. I'm just worried we'll forget about some specific case and break things by trying to be too clever.

Reply:
> we could use a regex to try to remove trailing 0s

Maybe this is the best option at the moment.

Reply:
@VaiTon @CharlesNepote : done, can you approve the changes? thanks!